Done with day 1 of AoC-2018

Joasan04 opened the pull request:
The first problem was to add toggether 972 numbers and.
The second problem was to find the first number that repeted when adding the numbers.
First i made a function to read the text of a file becuase it was to much text to put in an array.
Then i made the string in to a string array and then into
an int array becuase i needed to seperate the numbers so i could add them toggether.
Then for the second part of the problem i made a loop
that would see if the sum of the curent number matched any number that had come before.

Commit:
Done with day 1 of AoC-2018

The first problem was to add toggether 972 numbers and.
The second problem was to find the first number that repeted when adding the numbers.
First i made a function to read the text of a file becuase it was to much text to put in an array.
Then i made the string in to a string array and then into
an int array becuase i needed to seperate the numbers so i could add them toggether.
Then for the second part of the problem i made a loop
that would see if the sum of the curent number matched any number that had come before.

olssoj17855 commented on day-1/ChronalCalibration/ChronalCalibration/Program.cs:
```diff
@@ -0,0 +1,79 @@
+            var last = 0;
+            myList.Add(0);
+            var sum = 0;
+            while (true)
```
Looks like we could get stuck here forever! Should we perhaps define an upper limit for the number of iterations we do on the input data set?

olssoj17855 commented on day-1/ChronalCalibration/ChronalCalibration/Program.cs:
```diff
@@ -0,0 +1,79 @@
+    {
+        static void Main(string[] args)
+        {
+            var inputFilePath = @"C:\Users\joakim\repos\AoC-2018\day-1\ChronalCalibration\ChronalCalibration\Input.txt";
```
This file path does not seem to work for me :( Maybe there is no Joakim on my machine!

olssoj17855 commented on day-1/ChronalCalibration/ChronalCalibration/Program.cs:
```diff
@@ -0,0 +1,79 @@
+            var inputFilePath = @"C:\Users\joakim\repos\AoC-2018\day-1\ChronalCalibration\ChronalCalibration\Input.txt";
+            var numbersAsStringArray = ParseInput(inputFilePath);
+            var numbersAsIntArray = ToIntArray(numbersAsStringArray);
+            var Answer2 = FirstNumberTwice(numbersAsIntArray);
```
According to C# coding conventions, local variables start with small letters.

Commit:
New Feature: changeble number of iterations.

I added so you now can set the number of iterations,
that way it dosen't go on for ever if ther's an imposible senarion.
Added an error message if it couldn't find the answer to problem b.

Commit:
Relatve file path

The file path is now relative and will work for everyone.

olssoj17855 commented:
Everything seems resolved. Well done!
